assets/scripts/gallery/__tests__/Gallery.test.jsx

```diff
@@ -105,8 +105,8 @@ describe('Gallery', () => {
       }
     }
 
-    const wrapper = renderWithReduxAndIntl(<Gallery />, { initialState })
-    fireEvent.click(wrapper.container.querySelector('.gallery-shield'))
+    renderWithReduxAndIntl(<Gallery />, { initialState })
+    fireEvent.click(screen.getByTestId('gallery-shield'))
```

Comment:
I don't think screen has a way of using the query selector? In any case, RTL discourages that as practice. Since we don't have anything else in that container that the user sees, seems like a good case to use a test ID

Reply:
Normally I'd agree with changing the approach (whether with test ID or other methods) but I could also see a slippery slope where updating the render output to do test-ids (or other approaches) start making this issue much more complicated to finish.

For a quick fix, you should still be able to destructure the `container` from the render and call `querySelector`.

```js
const { container } = renderWithReduxAndIntl(<Gallery />, { initialState })
fireEvent.click(container.querySelector('.gallery-shield'))
```

Reply:
It seems to be working fine, though?

Reply:
Right. It works _fine_, I was just concerned that refactoring `container` usage now would complicate the completion of this issue. It's the reason why I didn't include it in the initial issue description.

That said, I also am generally not a fan of `test-id`s because they pollute the output HTML. Normally, we'd like to adopt something like this package, which can strip out the `testid`s in a production environment: https://github.com/coderas/babel-plugin-jsx-remove-data-test-id